modules/dnn/src/model.cpp

```diff
@@ -356,45 +357,58 @@ void DetectionModel::detect(InputArray frame, CV_OUT std::vector<int>& classIds,
                 height   = std::max(1, std::min(height, frameHeight - top));
 
                 predClassIds.push_back(classIdPoint.x);
-                predConf.push_back(static_cast<float>(conf));
+                predConfidences.push_back(static_cast<float>(conf));
                 predBoxes.emplace_back(left, top, width, height);
             }
         }
 
         if (nmsThreshold)
         {
-            std::map<int, std::vector<size_t> > class2indices;
-            for (size_t i = 0; i < predClassIds.size(); i++)
-            {
-                if (predConf[i] >= confThreshold)
+            if (!nmsDifferentLabels) {
```

Comment:
Keep origin code style - open a block in a new line

Reply:
Can you check the updated commit please?

Reply:
Same problem. Should be
```cpp
if (!nmsAcrossClasses)
{
```